mixins/common-mixin.html

```diff
@@ -182,8 +185,8 @@
     _setFitInto() {
       let ironDropdown = this._getIronDropdown();
       // fitInto element will not let the dropdown to overlap it's margins
-      if (!this.fitInto) {
-        this.fitInto = window.EtoolsEsmmFitIntoEl;
+      if (!this.fitInto && window.EtoolsEsmmFitIntoEl) {
+        ironDropdown.set('fitInto', window.EtoolsEsmmFitIntoEl);
```

Comment:
can be refactored, use else./
The 2 if's are mutually exclusive , but this is not reflected in code.